code/game/jobs/access.dm

```diff
@@ -243,7 +243,13 @@ var/obj/item/card/id/all_access/ghost_all_access
 		if(id)
 			return id
 	if(wear_id)
-		return wear_id.GetID()
+		var/id = wear_id.GetID()
+		if(id)
+			return id
```

Comment:
This is the same thing in more steps.

Reply:
It checks whether it has an ID. If it doesn't, it checks the next one. Returning early means the proc stops without checking the rest.